orderer/common/localconfig/config.go

```diff
@@ -282,8 +287,9 @@ var Defaults = TopLevel{
 
 // Load parses the orderer YAML file and environment, producing
 // a struct suitable for config use, returning error on failure.
-func Load() (*TopLevel, error) {
+func Load(enableCache bool) (*TopLevel, error) {
```

Comment:
I'd have expected the caching to be an exported package level var; since the test is ins the same package, it can set it to false from an init function or from TestMain.

Reply:
done, implemented an unexported function instead.